ansible/playbooks/roles/upgrade/tasks/kubernetes/upgrade-master0.yml

```diff
@@ -38,6 +38,12 @@
       delay: 30
       changed_when: false
 
+    - name: k8s/master0 | Add k8s annotation for containerd
+      delegate_to: "{{ groups.kubernetes_master[0] }}"
+      run_once: true
```

Comment:
Do we need this since this file is run only on master[0]?

```
- name: k8s | Upgrade first master to v{{ version }}
  include_tasks: kubernetes/upgrade-master0.yml
  when:
    - inventory_hostname == groups.kubernetes_master[0]
```